.circleci/config.yml

```diff
@@ -282,10 +329,25 @@ workflows:
           image_tag: heroku/pack:20
           image_tag_alias: heroku/pack:20
           requires:
+            - test-go-20
+            - test-java-20
+            - test-php-20
```

Comment:
Similarly, once the Ruby getting-started-guide addition is made above, Ruby will also need adding here

Reply:
fixed in 643440a